### ISSUE TITLE
Clef change size

### ISSUE THREAD

**Issue by craigsapp** [rism-digital/verovio#2849]
The size of clef changes is too small.  It is currently 66% of full-sized clefs, but should be increased to 75% of full-sized clefs.  Here is an example:

<img width="752" alt="Screen Shot 2022-05-08 at 10 01 32 AM" src="https://user-images.githubusercontent.com/3487289/167307028-01315906-2553-48f7-9d79-18da65593a63.png">

<details>
<summary>
Click to view MEI data for above example
</summary>

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/dev/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/dev/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="5.0.0-dev">
 <meiHead>
  <fileDesc>
   <titleStmt>
    <title />
   </titleStmt>
   <pubStmt />
  </fileDesc>
  <encodingDesc>
   <appInfo>
    <application isodate="2022-05-08T10:01:34" version="3.10.0-dev-2bebcfc-dirty">
     <name>Verovio</name>
     <p>Transcoded from Humdrum</p>
    </application>
   </appInfo>
  </encodingDesc>
  <workList>
   <work>
    <title xml:id="title-L9" analog="humdrum:Xfi" type="translated">extract -k 2</title>
   </work>
  </workList>
 </meiHead>
 <music>
  <body>
   <mdiv xml:id="mus49sl">
    <score xml:id="supah1">
     <scoreDef xml:id="s8ba9h2" midi.bpm="400.000000">
      <staffGrp xml:id="sc7zb3c">
       <staffDef xml:id="sb00wc3" n="1" lines="5">
        <clef xml:id="clef-L2F1" shape="F" line="4" />
       </staffDef>
      </staffGrp>
     </scoreDef>
     <section xml:id="section-L1F1">
      <measure xml:id="measure-L1" n="1">
       <staff xml:id="sehoiy1" n="1">
        <layer xml:id="layer-L1F1N1" n="1">
         <note xml:id="note-L4F1" dur="1" oct="4" pname="c" accid.ges="n" />
         <clef xml:id="clef-L5F1" shape="G" line="2" />
        </layer>
       </staff>
      </measure>
      <measure xml:id="measure-L6" right="end" n="2">
       <staff xml:id="staff-L6F1N1" n="1">
        <layer xml:id="layer-L6F1N1" n="1">
         <note xml:id="note-L7F1" dur="1" oct="4" pname="c" accid.ges="n" />
        </layer>
       </staff>
      </measure>
     </section>
    </score>
   </mdiv>
  </body>
 </music>
</mei>
```

</details>

67% should instead be reserved for cue-sized clef changes.

Measuring the size graphically:

<img width="764" alt="Screen Shot 2022-05-08 at 10 05 06 AM" src="https://user-images.githubusercontent.com/3487289/167307171-033fbdda-3921-427e-89fa-9e843e5f5729.png">

The problem most like occurs because of this statement on page 7 of *Behind Bars*:

<img width="395" alt="Screen Shot 2022-05-08 at 9 25 03 AM" src="https://user-images.githubusercontent.com/3487289/167307188-de48d914-bef0-49d4-8112-838e228ae870.png">

The problem is that the following examples  in the book all use 75% scaling instead of 67%.  Here is an example from page 8:

<img width="623" alt="Screen Shot 2022-05-08 at 10 06 43 AM" src="https://user-images.githubusercontent.com/3487289/167307234-393008a2-7324-4572-9ffd-a8fb965b074a.png">

And also, here is a brief survey of treble clef changes in late-nineteenth and twientieth century editions (including several computer-typeset editions):

<img width="28" alt="Screen Shot 2022-05-08 at 9 57 15 AM" src="https://user-images.githubusercontent.com/3487289/167307297-3662fadc-6675-406f-b359-30092aca7fa3.png"><img width="30" alt="Screen Shot 2022-05-08 at 9 59 27 AM" src="https://user-images.githubusercontent.com/3487289/167307291-56729e89-9172-4c48-8e14-d084f21f7c59.png"><img width="35" alt="Screen Shot 2022-05-08 at 9 59 00 AM" src="https://user-images.githubusercontent.com/3487289/167307293-05c4f2cc-3842-41f3-aec5-ac760bb2aaff.png"><img width="30" alt="Screen Shot 2022-05-08 at 9 58 41 AM" src="https://user-images.githubusercontent.com/3487289/167307294-0f27c686-c8fa-484d-959f-c458a11252b0.png"><img width="33" alt="Screen Shot 2022-05-08 at 9 58 22 AM" src="https://user-images.githubusercontent.com/3487289/167307295-ad0bde83-8643-4457-ab7b-75a23f5ef102.png"><img width="21" alt="Screen Shot 2022-05-08 at 9 57 55 AM" src="https://user-images.githubusercontent.com/3487289/167307296-e35ed7c0-5e5c-4fb4-afe6-e98082a6e58d.png"><img width="30" alt="Screen Shot 2022-05-08 at 9 56 48 AM" src="https://user-images.githubusercontent.com/3487289/167307303-df615e42-50e9-429e-a455-e5df6188b596.png"><img width="26" alt="Screen Shot 2022-05-08 at 9 55 45 AM" src="https://user-images.githubusercontent.com/3487289/167307305-1f9c66b8-d0a5-40ca-bc05-fe0f42aca2eb.png"><img width="39" alt="Screen Shot 2022-05-08 at 9 55 30 AM" src="https://user-images.githubusercontent.com/3487289/167307306-7999366d-3488-479b-bb29-2986889a85f5.png"><img width="45" alt="Screen Shot 2022-05-08 at 9 54 56 AM" src="https://user-images.githubusercontent.com/3487289/167307307-5671e333-e9ed-4073-aa9a-a629403b717b.png"><img width="40" alt="Screen Shot 2022-05-08 at 9 53 45 AM" src="https://user-images.githubusercontent.com/3487289/167307309-65679667-8f9f-4d54-ac13-28e5dfa9ddac.png">

All are using 75%. As a comparison, here is verovio which is using 67%:

<img width="59" alt="Screen Shot 2022-05-08 at 9 59 35 AM" src="https://user-images.githubusercontent.com/3487289/167307289-5bc9bd4b-1d4f-4c67-be13-be998acbfc41.png">

**Comment by rettinghaus** [comment]
So why not just use `--clef-change-factor`?

**Comment by craigsapp** [comment]
Because 75% is 100% correct.

**Comment by craigsapp** [comment]
Was last correct in version 3.2:

<img width="1336" alt="Screen Shot 2022-05-08 at 12 40 52 PM" src="https://user-images.githubusercontent.com/3487289/167312952-bbdde584-328b-4c3a-a508-617605b2a092.png">

**Comment by lpugin** [comment]
From @craigsapp's examples and considering the incoherence in Behind Bars it looks like it would make sense to revert  the default to 75%. Finale also has a default of 75%, and so seems to have LilyPond - looking at some scores. Only Dorico has a default of 2/3. But as pointed out by @rettinghaus we have now the option `--clef-change-factor` which can be used to set it to any less common value.

**Comment by craigsapp** [comment]
> Only Dorico has a default of 2/3.

As Dorico is developed by English programmers, I suspect that they used the 2/3 value mentioned in *Behind Bars*, which is obviously a mistake since it does not match the example.  The 66% clef looks too small to me, which is why I checked into it and noticed the inconsistency in the *Behind Bars* description versus examples.  And 11/11 of the editions I looked at from a variety of publishers emphasize the point since they were all at 75%.

-=+Craig

**Comment by rettinghaus** [comment]
It seems that @craigsapp is pretty confident with his 75%, but in the scores I checked it is more like 80%. Also in classic printed material often G and F clefs don't show the same ratio, with F clefs scaled larger, more at approx. 90%. 
I went with Gould here because she gives a precise number (although I also think that this is too small). 
Another possibility would be to use the small clefs coming with the SMuFL font, which maybe would bring a more consistent look. But then we probably would give up some flexibility we have right now, e.g. with ottava clefs. I've never seen the latter as changing clefs in real life, but maybe someone has an example by hand? 
On the other hand, in early modern prints changing clefs were not printed smaller, so going with the standard size (glyphs) seems to be fine when handling mensural notation/clefs. 
One last point: thinking about this, I would expect that when selecting a specific clef (by `@glyph.name` or similar) Verovio would show this clef at 100% size, but right now it gets scaled down (when it's within a layer). A per clef scaling mechanism could be implemented by adding support on `@fontsize`. Any opinion on this?
